docker/images/skycoindev-cli/Dockerfile

```diff
@@ -0,0 +1,61 @@
+# Creates an image for skycoin development
```

Comment:
I'd recommend to move this `./docker/images/skycoindev-cli/Dockerfile` => `./docker/images/dev-cli/Dockerfile`